app/views/layouts/application.html.erb

```diff
@@ -5,8 +5,8 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application" %>
-    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
```

Comment:
This is a different version of bootstrap than is pinned in the importmap. Is that OK?

Reply:
Yeah, it works fine.

Reply:
Is there any reason to want diff versions of bootstrap? I.e., any reason not to bump the js pin to 4.6.1?

Reply:
As a follow up I was going to upgrade to 5.0. But no, there's no reason to have different versions.

Reply:
OK, no bigs.